spec/locale-sensitive-functions.html

```diff
@@ -65,27 +65,23 @@ <h1>String.prototype.toLocaleLowerCase ( [ _locales_ ] )</h1>
         1. Else,
           1. Let _requestedLocale_ be DefaultLocale().
         1. Let _noExtensionsLocale_ be the String value that is _requestedLocale_ with all Unicode locale extension sequences (<emu-xref href="#sec-unicode-locale-extension-sequences"></emu-xref>) removed.
-        1. Let _availableLocales_ be a List with the language tags of the languages for which the Unicode character database contains language sensitive case mappings.
+        1. Let _availableLocales_ be a List with language tags that includes the languages for which the Unicode Character Database contains language sensitive case mappings. Implementations may add additional language tags if they support custom tailoring.
```

Comment:
I'd delete " if they support custom tailoring". The phrasing I'd suggest, and which much of the spec tends towards, implementations maintain their own locale database, rather than talking about "tailoring".

Reply:
@leobalter also thought I should use language more like what you just suggested, @littledan -- but I only see the word "database" used in the spec when referring to "Unicode Character Database" or "IANA Time Zone Database". I feel like I would be confused if I saw a reference suddenly to a "implementation maintained locale database". But then again, I am new to reading this spec :)

What about not using the phrase "custom tailoring":
`Implementations may add additional language tags if they support case mapping for additional locales.`